source/assets/sass/components/_buttons.scss

```diff
@@ -39,11 +39,11 @@
 
     &:hover,
     &:focus {
-      background-color: var(--action);
+      --color-action-bg: var(--action);
```

Comment:
I think ideally, when `.sl-c-button` is added to a link, that should only have to override the base definition of `background`, and then the local button variables should work without conflict.

To make that happen, I think you would need to use a similar pattern for links, so on link hover/focus we update e.g. `--color-link-bg` rather than updating `background`. Then here we change the `background` to use the button vars instead of the link vars - and we can update those button vars without any external conflict.

Reply:
@mirisuzanne cleaned this up in https://github.com/oddbird/sass-site/pull/10/commits/2452f354b5d04f7c2b87b3b9f99b1eebf1484bf9